pkg/v1/config/defaults.go

```diff
@@ -37,3 +53,24 @@ var DefaultRepositories []configv1alpha1.PluginRepository = []configv1alpha1.Plu
 		GCPPluginRepository: &CoreGCPBucketRepository,
 	},
 }
+
+// DefaultStandaloneDiscoveryImage returns the default Standalone Discovery image
+// from the configured build time variables
+func DefaultStandaloneDiscoveryImage() string {
+	defaultStandaloneDiscoveryRepository := DefaultStandaloneDiscoveryRepository
+	defaultStandaloneDiscoveryImagePath := DefaultStandaloneDiscoveryImagePath
+	defaultStandaloneDiscoveryImageTag := DefaultStandaloneDiscoveryImageTag
+
+	// Run-time overrides of the configuration
+	if customImageRepo := os.Getenv(constants.ConfigVariableCustomImageRepository); customImageRepo != "" {
+		defaultStandaloneDiscoveryRepository = customImageRepo
+	}
+	if imagePath := os.Getenv(constants.ConfigVariableDefaultStandaloneDiscoveryImagePath); imagePath != "" {
+		defaultStandaloneDiscoveryImagePath = imagePath
+	}
+	if imageTag := os.Getenv(constants.ConfigVariableDefaultStandaloneDiscoveryImageTag); imageTag != "" {
+		defaultStandaloneDiscoveryImageTag = imageTag
+	}
```

Comment:
All these `if` conditions seem to eval to false when running unit tests, so they're only "half" covered.

Reply:
added unit tests